### DISCUSSION TITLE
Add ddvk's USB-OTG recovery manual

### DISCUSSION THREAD

**Discussion by eloeffler-fhnw** [danielebruneo/remarkable2-hacks#2] [Ideas]
Hi, thanks for setting this up!

As you ask in the repo to talk about any contributions etc. in here, this is my suggestion :)

It would be great if you could add ddvk's USB-OTG recovery instructions. 
It takes a lot of effort but it works and gave me back access to my rM2.
(I followed it successfully without getting hold of the pogo pins, with bent wire and a hardpaper-board.)

The instructions are here:
https://github.com/ddvk/remarkable2-recovery

**Comment by danielebruneo** [comment]
That's a great suggestion! 
Thanks.
